### ISSUE TITLE
Error with Custom Object Not Making It Back Through StackStorm CLI

### ISSUE THREAD

**Issue by chavenor** [StackStorm-Exchange/stackstorm-digitalocean#2]
We tested my key on seperate StackStorm systems.    No results were present for any of the API calls.

**Comment by chavenor** [comment]
```python 
cat << "EOF" > /opt/stackstorm/packs/digitalocean/actions/get_all_droplets.py
import digitalocean
from st2actions.runners.pythonrunner import Action


class GetAllDropletsAction(Action):

    def __init__(self, config):
        super(GetAllDropletsAction, self).__init__(config)
        self._token = config['token']

    def run(self, **kwargs):
        manager = digitalocean.Manager(token=self._token)
        return manager.get_all_droplets()
EOF
```

**Comment by chavenor** [comment]
The results from the API package need to be modified to standard python objects.

**Comment by nmaludy** [comment]
## Investigation
The `get_all_droplets` action is completing successfully, however returning no results.

I then created a simply python test to confirm that the token we were using was correct and had valid permissions:
```
import digitalocean
manager = digitalocean.Manager(token="secretspecialuniquesnowflake")
my_droplets = manager.get_all_droplets()
print(my_droplets)
```

This printed:
```
[<Droplet: 46166796 sendy>]
```

As you can see the API call is returning a list of `digitalocean.Droplet` objects, and when returned from the action StackStorm doesn't know what to do with them, so they're dropped.

## Solution
Convert the `digitalocean.Droplet` objects into lists and dicts so that StackStorm can work with them.

See the following PR for my first shot at code that fixes this: https://github.com/StackStorm-Exchange/stackstorm-digitalocean/pull/3

**Comment by LindsayHill** [comment]
Resolved by #3
